src/packages/pickerview/doc.md

```
@@ -0,0 +1,94 @@
+
+| 属性 | 说明 | 类型 | 默认值 |
+| --- | --- | --- | --- |
+| label | 选项的文字内容 | `string` \| `number` | `-` |
+| value | 选项对应的值，且唯一 | `string` \| `number` | `-` |
+| children | 用于级联选项 | `PickerOptionItem[]` | `-` |
+
+## 主题定制
+
+### 样式变量
+
+组件提供了下列 CSS 变量，可用于自定义样式，使用方法请参考 [ConfigProvider 组件](#/zh-CN/component/configprovider)。
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)